articles/what-is-self-supervised-learning/index.md

```diff
@@ -0,0 +1,96 @@
+
+This multifaced approach to learning has been so easy for humans. But, that has not been the case for deep learning systems. Though we have seen robust deep learning systems that have performed image recognition and natural language processing tasks, performing complex tasks has remained challenging. This is the problem self-supervised learning is trying to address. 
+
+Self-supervised learning has recently been a hot topic in Artificial Intelligence. It is a form of supervised learning that doesn't require human input to perform data labeling. It can also be viewed as a branch of unsupervised learning. The results are obtained by models that analyze data, label and categorize information independently without any human input. The only difference is that, unlike unsupervised learning, self-supervised learning does not perform the grouping and clustering of data, as is the case with unsupervised learning. 
+
+This learning type allows machines to examine part of a data example to figure out the remaining part. In simple terms, self-supervised learning learns from unlabeled data to fill in the blanks for missing pieces. These data can be in the form of images, text, audio, and videos. 
+For example, in videos, the machine can predict the missing part of a video given only a video section. Videos can also use it in predicting missing frames in a video.
+
+Self-supervised learning aims to make deep learning models data-efficient. This means that it helps reduce the over-dependence on vast amounts of data to achieve good models.
+
+### Applications of Self-Supervised Learning
+
+1. Natural Language Processing
+
+Self-supervised learning helps predict the missing words in a text in the field of NLP. This is achieved by showing segments of texts to a giant neural network with billions of parameters, i.e., the likes of OpenAI's [GPT-3](https://openai.com/blog/openai-api/) and Google's [BERT](https://arxiv.org/pdf/1810.04805.pdf). You mask 15% of the text to force the network to predict the pieces of words that are missing. 
+
+2. Computer Vision
+
+[SimCLR](https://arxiv.org/pdf/2002.05709.pdf) is a framework used to learn visual representations in images using self-supervised learning. The framework performs two main tasks: a pretext task and the downstream (real) task. Self-supervised learning is used in the pretext task. It involves performing simple augmentation tasks such as random cropping, random color distortions, and random Gaussian blur on input images. This process enables the model to learn better representations of the input images. These results are passed into the downstream task module, which performs the main tasks such as detection and classification tasks.
```

Comment:
Perhaps briefly define a random Gaussian blur for a reader who might be unaware.